_data/oh1addons_infos.yml

```diff
@@ -161,6 +161,10 @@
   description: This binding allows openHAB to act as an MQTT client, so that openHAB items can send and receive MQTT messages to/from an MQTT broker
   wiki_url: https://github.com/openhab/openhab/wiki/MQTT-Binding
 
+- label: Owntracks Binding
+  description: Owntracks (formerly known as Mqttitude) provides device location data via an mqtt bridge to openHAB
```

Comment:
It sort of doesn't matter, but it was called MQTTitude before OwnTracks (note differences in capitalization).  Also mqtt would be better as MQTT.